styles.css

```diff
@@ -328,78 +170,29 @@ body {
 
   .menuitems {
     color: white;
+    margin-right: 0px;
     text-align: center;
     cursor: pointer;
   }
 
-  .dropdown {
-    position: relative;
-    display: inline-block;
-  }
-
-  .dropdown-content {
-    display: none;
-    position: absolute;
-    background-color: #f9f9f9;
-    min-width: 160px;
-    box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
-    z-index: 1;
-  }
-
-  .dropdown-content a {
-    color: black;
-    padding: 12px 16px;
-    text-decoration: none;
-    display: block;
-  }
-
-  .dropdown-content a:hover {
-    background-color: #f1f1f1
-  }
-
-  .dropdown:hover .dropdown-content {
-    display: block;
-  }
-
-  .dropdown:hover .dropbtn {
-    background-color: #3e8e41;
-  }
-
   .grid-containerkart {
-    display: grid;
     grid-template-rows: auto auto;
+    grid-template-columns:none;
   }
 
   .kart {
-    cursor: pointer;
     margin: 0;
     position: absolute;
     top: 50%;
     left: 50%;
     -ms-transform: translate(-50%, -50%);
     transform: translate(-50%, -50%);
-    cursor: pointer;
   }
-
-  .arrow {
-    width: 15px;
-    margin-bottom: -5px;
-  }
-
   .arrowbuttonnav {
-    background-color: #7ba435;
-    border: none;
-    color: white;
     padding: 5px;
     padding-right: 5%;
     padding-left: 5%;
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 16px;
     margin: 4px 2px;
-    cursor: pointer;
-    border-radius: 10px;
     margin: 0;
```

Comment:
margin: 0; will rewrite the styles specified above

Reply:
Updated the CSS.